releases/release-1.26/release-notes/sessions/maps-1663864374.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mail": "rodomar@outlook.com",
+  "name": "Rodolfo Martínez Vega",
+  "date": 1663864374,
+  "prs": [
+    {
+      "nr": 112526,
+      "hash": "6a379206ecb59226d5032d9c8826fa0f8b93f272"
+    },
+    {
+      "nr": 112046,
+      "hash": "2fd5c7e2afd8f868da8372210638447456941ae8"
+    },
+    {
+      "nr": 112309,
```

Comment:
Any idea why this file contains one more item here than the one with maps ?

Reply:
Good catch! It seems that I entered `n` exactly that PR when `krel` asked me if I wanted to edit the release note:

![image](https://user-images.githubusercontent.com/2797052/191895196-907ff86a-ec13-4a9a-9a5a-f7ddde97e898.png)

So, the next time I want to pause the process there, I should do a `Ctrl+C` immediately and create the PR then (instead of entering `n` first)

I'll update this PR to remove this hash at the map file and do it in another PR :)